<ui version="4.0" >
 <class>StatusWidget</class>
 <widget class="QWidget" name="StatusWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>37</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>StatusWidget</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>6</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelNumTotalText" >
     <property name="text" >
      <string>Total:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumTotal" >
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumSelectedText" >
     <property name="text" >
      <string>Selected:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumSelected" >
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumRunText" >
     <property name="text" >
      <string>Run:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelNumRun" >
     <property name="text" >
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumSuccessPic" >
       <property name="toolTip" >
        <string>Success</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/success.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumSuccess" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumExceptionsPic" >
       <property name="toolTip" >
        <string>Exceptions</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/exception.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumExceptions" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumFatalsPic" >
       <property name="toolTip" >
        <string>Fatals</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/fatal.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumFatals" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumErrorsPic" >
       <property name="toolTip" >
        <string>Errors</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/error.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumErrors" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumWarningsPic" >
       <property name="toolTip" >
        <string>Warnings</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/warning.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumWarnings" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>3</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelNumInfosPic" >
       <property name="toolTip" >
        <string>Infos</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" >:/icons/info.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumInfos" >
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressRun" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16</height>
      </size>
     </property>
     <property name="value" >
      <number>0</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../../../Desktop/gsoc/qxrunner.0.9.2/qxrunner/src/qxrunner/resources/qxrunner.qrc" />
 </resources>
 <connections/>
</ui>
